pages/common/az.md

```diff
@@ -0,0 +1,27 @@
+# az
+
+> The official CLI tool for Microsoft Azure.
+
+- Log in to Azure:
+
+`az login`
+
+- Manage azure subscription information:
+
+`az account`
+
+- List all Azure Managed Disks:
```

Comment:
```suggestion
- List all Azure managed disks:
```

Reply:
These are product names, and are capitalised that way in official documentation: https://docs.microsoft.com/en-us/azure/virtual-machines/windows/managed-disks-overview

Reply:
ohh.. my bad! LGTM then

Reply:
Yes; It is the case in all examples.